CONFIG/config.cpp

```diff
@@ -0,0 +1,436 @@
+#include "config.h"
+
+#include "ConfigCommandLineInfo.h"
+#include "MainDlg.h"
+#include "detectdx5.h"
+
+#include <direct.h> // _chdir
+#include <mxdirectx/mxdirect3d.h>
+#include <process.h> // _spawnl
+
+// FIXME: are these tags correct?
+// FIXME: how to tag static in-method variables?
```

Comment:
@disinvite

Reply:
These should go in config.h:
```
// FUNCTION: CONFIG 0x402c30
// CConfigApp::GetMessageMap

// GLOBAL: CONFIG 0x406008
// CConfigApp::messageMap

// GLOBAL: CONFIG 0x406010
// CConfigApp::_messageEntries
```

Reply:
Thanks applied this to all CWnd subclasses.